### PR TITLE
[Refactoring] - Added Data Types tests

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java [hunk 1]
```diff
@@ -62,14 +65,28 @@ public static SelenideElement getDataTypeWithName(String datatypeName) {
...
-    public static void createDataType(String name, String description, Boolean isRest) {
+    public static void createDataType(String name, String description, String example, Boolean isRest) {
         log.info("Creating data type with name {} and description {} and rest resources are {}", name, description, isRest);
 
-        CommonUtils.getLabelWithName("name", CommonUtils.getAppRoot()).setValue(name);
+        SelenideElement entityForm = CommonUtils.getAppRoot().$(MainPageElements.ENTITY_EDITOR_FORM);
+
+        CommonUtils.getLabelWithName("name", entityForm).setValue(name);
+
+        if (!description.isEmpty()) {
```
Comment (mcada) [inline src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java]: If description/example parameters are optional, would it make sense to overload the method for easier use? These ifs don't look good. I understand the usage, but isn't there a better way how to do it?

File: src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java [hunk 2]
```diff
@@ -62,14 +65,28 @@ public static SelenideElement getDataTypeWithName(String datatypeName) {
...
+            entityForm.$(By.id("description")).$("ace-editor textarea").sendKeys(description);
```
Comment (mcada) [inline src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java]: Can we add `ace-editor textarea`  selector to `Elements.` inner class?

Comment (mmajerni) [reply src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java]: added

File: src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java (hunk 1 above)
Comment (mmajerni) [reply src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java]: Added docs as we discussed

**Pull request merged**